README.md

```diff
@@ -1,53 +1,56 @@
-# GeoArrow - geospatial data with Apache Arrow
 
-Specifications for storing geospatial data in Apache Arrow.
+# GeoArrow Specification
+
+This repository contains specifications for storing geospatial data in Apache Arrow
+and Arrow-compatible data structures and formats.
 
 The [Apache Arrow](https://arrow.apache.org/) project specifies a standardized
 language-independent columnar memory format. It enables shared computational libraries,
-zero-copy shared memory and streaming messaging, interprocess communication, etc and is
+zero-copy shared memory and streaming messaging, interprocess communication, and is
 supported by many programming languages and data libraries.
 
 Spatial information can be represented as a collection of discrete objects using points,
 lines and polygons (i.e., vector data). The
 [Simple Feature Access](https://www.ogc.org/standards/sfa) standard provides a widely
 used abstraction, defining a set of geometries: Point, LineString, Polygon, MultiPoint,
-MultiLineString, MultiPolygon, GeometryCollection. Next to a geometry, simple features
+MultiLineString, MultiPolygon, and GeometryCollection. Next to a geometry, simple features
 can also have non-spatial attributes that describe the feature.
 
 Geospatial data often comes in tabular format, with one or more columns with
 feature geometries and additional columns with feature attributes. The Arrow columnar
-memory model is therefore perfectly suited to store such data, both vector features and
+memory model is well-suited to store both vector features and
 their attribute data. The GeoArrow specification defines how the vector features
 (geometries) can be stored in Arrow (and Arrow-compatible) data structures.
 
 This repository contains the specifications for:
 
-- The memory layout for storing geometries in an Arrow field ([format.md](./format.md))
-- The Arrow extension type definitions ([extension-types.md](./extension-types.md))
+- The memory layout for storing geometries in an Arrow array ([format.md](./format.md))
+- The Arrow extension type definitions that ensure type-level metadata (e.g., CRS) is
+  propagated when used in Arrow implementations ([extension-types.md](./extension-types.md))
 
 Defining a standard and efficient way to store geospatial data in the Arrow memory
-layout helps interoperability between different tools and ensures geospatial tools can
+layout enables interoperability between different tools and ensures geospatial tools can
 leverage the growing Apache Arrow ecosystem:
 
 - Efficient, columnar file formats. Leveraging the performant and compact storage of
   Apache Parquet as a vector data format in geospatial tools using
-  [GeoParquet](https://github.com/opengeospatial/geoparquet/).
+  [GeoParquet](https://github.com/opengeospatial/geoparquet/)
```

Comment:
I'd go the other way and add a period after each bullet point, but this is minor 🤷‍♂️